Automattic/course
# Update typography settings

Resolve part of #3 

## Introduced changes
* Add [League Gothic](https://fonts.google.com/specimen/League+Gothic) as Webfont
* Add [EB Garamond](https://fonts.google.com/specimen/EB+Garamond) as Webfont
* Set h1-h6 + paragraph  font sizes using the fluid strategy

## Extra information
This PR is not using SF font family described on the h2-h4 and on the paragraph because the SF family apparently is not license-friendly to be used on the web and the file size, even compressed is terrible > 2mb.
I opened a thread p1664919932124569-slack-C013QUH20TS  to discuss it.

Considering the number of changes it is introducing on the same file, I would like to recommend merging it while we discuss changing the font family. 

### Testing instructions
* Create a page
* Past the below code on the post content
* Check the font size using the `Computed`  tab on the Chrome Developer Tools

<img width="602" alt="Screen Shot 2022-10-04 at 17 22 38" src="https://user-images.githubusercontent.com/38718/193918991-a49a7295-34cf-4cc3-89d8-70ab01809a30.png">

```

<h1>I am THE H1 TAG</h1>



<h2>I am the H2 tag</h2>



<h3>I am the H3</h3>



<h4>I am the H4</h4>



<h4>I am the h5</h4>



<h6>I am the h6</h6>



<p>This blog is run by Professor Jean Biche, Chair of Art History at Sacré-Cœur Université Paris. He joined the Department of Art in 2009 moving from the University of Notredame where he was a Professor of Global History and Culture.</p>

```


## Screenshots

### Desktop
<img width="751" alt="Screen Shot 2022-10-04 at 17 14 44" src="https://user-images.githubusercontent.com/38718/193917375-60767db6-4a43-49b3-8dbb-7866cdb6b3f2.png">

### Mobile
<img width="352" alt="Screen Shot 2022-10-04 at 17 16 01" src="https://user-images.githubusercontent.com/38718/193917626-9c27a02e-7909-4b94-9693-074db6a0f2bc.png">
